src/components/BottomBarTabView.vue

```diff
@@ -0,0 +1,239 @@
+        <bottombartab
+          v-bind="bottomCourse"
+          :id="bottomCourse.id"
+          :subject="bottomCourse.subject"
+          :number="bottomCourse.number"
+          :color="bottomCourse.color"
+          :courseObj ="bottomCourse"
+          :isFirstTab = true
+          :isExpanded ="isExpanded"
+          @bottomBarTabToggle="bottomBarTabToggle"
+          @deleteBottomTab="deleteBottomTab"
+          @toggleFromTab="toggleFromTab"
+        />
+      </div>
+      <div v-for="bottomCourse in bottomCourses.slice(1)" :key="bottomCourse.id" class="bottombartabview-courseWrapper">
+        <bottombartab
+          v-bind="bottomCourse"
+          :id="bottomCourse.id"
+          :subject="bottomCourse.subject"
+          :number="bottomCourse.number"
+          :color="bottomCourse.color"
+          :courseObj ="bottomCourse"
+          :isFirstTab = false
+          :isExpanded ="isExpanded"
+          @bottomBarTabToggle="bottomBarTabToggle"
+          @deleteBottomTab="deleteBottomTab"
+          @toggleFromTab="toggleFromTab"
+        />
+      </div>
+    </div>
+    <div v-if="seeMoreCourses.length > 0" class="bottombartabview-seeMoreWrapper">
+      <div class="bottombarSeeMoreTab" @click="bottomBarSeeMoreToggle">
+          <div class="bottombarSeeMoreTab-name">{{seeMoreString}}</div>
+          <img v-if="!seeMoreOpen" class="bottombarSeeMoreTab-arrow" src="@/assets/images/uparrow-white.svg" />
+          <img v-if="seeMoreOpen" class="bottombarSeeMoreTab-arrow" src="@/assets/images/downarrow-white.svg" />
+      </div>
+
+      <div v-if="seeMoreOpen" class="bottombarSeeMoreOptions">
+        <div class="seeMoreCourse-content">
+            <div
+              v-for="seeMoreCourse in seeMoreCourses"
+              :key="seeMoreCourse.id"
+              class="seeMoreCourse-option"
+            >
+                <span class="seeMoreCourse-option-text" @click="moveToBottomBar(seeMoreCourse)">{{ seeMoreCourse.subject }} {{ seeMoreCourse.number}}</span>
+                <img class="seeMoreCourse-option-delete" src="@/assets/images/x-blue.svg" @click="deleteSeeMoreCourse(seeMoreCourse)"/>
+            </div>
+        </div>
+      </div>
+
+    </div>
+
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import BottomBarTab from '@/components/BottomBarTab';
+
+Vue.component('bottombartab', BottomBarTab);
+
+export default {
+  data() {
+    return {
+      seeMoreOpen: false
+    };
+  },
+  props: {
+    bottomCourses: Array,
+    seeMoreCourses: Array,
+    isExpanded: Boolean
+  },
+
+  computed: {
+    seeMoreString() {
+      return 'See More';
+    }
+  },
+
+  methods: {
+    bottomBarTabToggle(courseObj) {
+      this.$emit('bottomBarTabToggle', courseObj);
+    },
+
+    deleteBottomTab(subject, number) {
+      for (let i = 0; i < this.bottomCourses.length; i += 1) {
+        if (this.bottomCourses[i].subject === subject && this.bottomCourses[i].number === number) {
```

Comment:
This code relies on there only being one version of a course in the expanded view of the course. Is this a problem if we are now fine with allowing people to have multiple of the same course? Especially if they are different colors?

Reply:
At the moment, no dupe courses are allowed for bottom tab including see more. If a user does have multiples of the same course (say CS 2110) and they click on the course card again, if CS 2110 is already a bottom bar tab, it is brought to the very front (with the color of that clicked course card). I don't think it would be very useful to have dupe bottom tabs even though they can have dupe course cards since the expanded bottom bar course details would be the same. Since there would only be one ver of a course, my delete functionality relied on this. Or are we saying that expanded course details could differ for the same course?

Reply:
If we want course identification to be more unique, we could assign/use a unique id prop?

Reply:
Discussed over slack - Want a unique id integer that starts at 0, and increases every time a course is added, so then there is never a duplicate. We can store and retrieve it from firebase per user, so that it always increases for a single user. Can be done in a separate PR, because it functionally works fine for bottom bar, but not across the rest of the platform.